DXRando/DeusEx/Classes/WHPiano.uc

```diff
@@ -436,6 +295,355 @@ function string GetSongMessage(Sound SelectedSound)
+            break;
+        case 45:
+            SelectedSound = sound'REMansionBasement';
+            duration = 9;
+            break;
+        case 46:
+            SelectedSound = sound'PachelbelsCanon';
+            duration = 9;
+            break;
+        case 47:
+            SelectedSound = sound'SMRPGForestMaze';
+            duration = 5;
+            break;
+        case 48:
+            SelectedSound = sound'HKSynapse';
+            duration = 5;
+            break;
+        case 49:
+            SelectedSound = sound'ToZanarkand';
+            duration = 8;
+            break;
+        case 50:
+            SelectedSound = sound'BubbleBobble';
+            duration = 8;
+            break;
+        case 51:
+            SelectedSound = sound'CruelAngelsThesis';
+            duration = 8;
+            break;
+        case 52:
+            SelectedSound = sound'ZeldaOverworld';
+            duration = 6;
+            break;
+        case 53:
+            SelectedSound = sound'Terran1';
+            duration = 7;
+            break;
+        case 54:
+            SelectedSound = sound'BabaYetu';
+            duration = 11;
+            break;
+        case 55:
+            SelectedSound = sound'DuckTalesMoon';
+            duration = 9;
+            break;
+        case 56:
+            SelectedSound = sound'HKHornet';
+            duration = 5;
+            break;
+        case 57:
+            SelectedSound = sound'THPSSuperman';
+            duration = 10;
+            break;
+        case 58:
+            SelectedSound = sound'BakaMitai';
+            duration = 10;
+            break;
+        case 59:
+            SelectedSound = sound'DragonRoostIsland';
+            duration = 7;
+            break;
+        case 60:
+            SelectedSound = sound'LonelyRollingStar';
+            duration = 11;
+            break;
+        case 61:
+            SelectedSound = sound'MM3WeaponGet';
+            duration = 4;
+            break;
+        case 62:
+            SelectedSound = sound'SOTNTragicPrince';
+            duration = 7;
+            break;
+        case 63:
+            SelectedSound = sound'PictionaryDrawingGame';
+            duration = 10;
+            break;
+        case 64:
+            SelectedSound = sound'CelesteFirstSteps';
+            duration = 7;
+            break;
+        case 65:
+            SelectedSound = sound'MM2WilyStage1';
+            duration = 8;
+            break;
+        case 66:
+            SelectedSound = sound'Mario2Overworld';
+            duration = 10;
+            break;
+        case 67:
+            SelectedSound = sound'SMGCometObservatory';
+            duration = 8;
+            break;
+        default:
+            SelectedSound = None;
+            duration = 0;
+            log("DXRPiano went too far this time!  Got "$currentSong);
+    }
+}
+
+//Old logic that purely picked a random song (except for the last played one)
+function int PickSongIndexOld()
```

Comment:
do we still need this?

Reply:
Probably not - just seemed worth throwing to the side and not deleting immediately